src/main/java/com/cs56fitnessapp/models/Meal.java

```diff
@@ -1,29 +1,91 @@
 package com.cs56fitnessapp.models;
+import java.util.ArrayList;
+// import java.util.Date;
+import java.time.LocalDate;
+
 
 /**
- * @author Nerissa Hsieh
+ * @author Nerissa Hsieh & Jimwell Castillo
  * Date Created: 10/03/17
- * Date Modified: 10/03/17
+ * Date Modified: 10/20/17
  */
 
-public class Meal implements DietFacts {
+public class Meal implements DietFacts{
+
+      //date
+    private LocalDate date;
+    private String title;
+    //foodList
+    private ArrayList <FoodEntry> foodList;
+    private String MealTitle;
```

Comment:
By convention variables names as well as data field names must be camel case starting with lowercase letter.
Correct naming is `private String mealTitle;`